### ISSUE TITLE
Create Pull Request Template

### ISSUE THREAD

**Issue by robertkossendey** [MrPowers/mack#92]
We should create a template for pull requests, requiring the contributor to tick a box stating that he read the contributing guidelines (CONTRIBUTING.md). By that we can make sure that the contributor has run the pre-commit hooks on his machine (flake8, black and pytest).

**Comment by MrPowers** [comment]
Yes, I like this idea, thank you for making this suggestion.  Would be great for someone to grab this.

**Comment by robertkossendey** [comment]
@MrPowers we should also add an Issue Template with "I would like to contribute" checks as well. More and more people are opening issues / contributing, so this will come in handy.

**Comment by MrPowers** [comment]
@robertkossendey - yea, an issue template with "I would like to contribute" would be great as well.  Good idea!

**Comment by Triamus** [comment]
Would be great if pure documentation commits don't require pre-commit setup as it may scare off folks who otherwise have valuable contributions. Can you exclude changes to the README.md from pre-commit hooks requirement by default somehow?

**Comment by Oguzozcn** [comment]
Hi, Can I take this?

**Comment by paulooctavio** [comment]
@MrPowers Can I take this one? It will be my first issue, so I might just need a little guidance.
